<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jess Scrimshire">
<meta name="dcterms.date" content="2024-02-14">

<title>Impact of novel Alzheimer’s disease drug discovery on the research field using text mining and topic models</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="report_files/libs/clipboard/clipboard.min.js"></script>
<script src="report_files/libs/quarto-html/quarto.js"></script>
<script src="report_files/libs/quarto-html/popper.min.js"></script>
<script src="report_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="report_files/libs/quarto-html/anchor.min.js"></script>
<link href="report_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="report_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="report_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="report_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="report_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="report_files/libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="report_files/libs/wordcloud2-0.0.1/wordcloud.css" rel="stylesheet">
<script src="report_files/libs/wordcloud2-0.0.1/wordcloud2-all.js"></script>
<script src="report_files/libs/wordcloud2-0.0.1/hover.js"></script>
<script src="report_files/libs/wordcloud2-binding-0.2.1/wordcloud2.js"></script>

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Impact of novel Alzheimer’s disease drug discovery on the research field using text mining and topic models</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Jess Scrimshire </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">February 14, 2024</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<section id="abstract" class="level1">
<h1>Abstract</h1>
<p>Alzheimer’s disease (AD) is a chronic neurodegenerative disease affecting over 55 million people worldwide. Currently there are no treatments that cure AD, however recently the anti-amyloid immunotherapy lecanemab has been granted accelerated approval by the FDA. Every year thousands of scientific articles are published concerning AD, however it can be time consuming and laborious to disseminate this information into a systematic review whilst highlighting current research. Full abstract text from PubMed for documents containing the MeSH term “Alzheimer’s Disease” were tokenised, cleaned then allocated to either a “pre-leca” or “post-leca” corpus relative to the date of lecanemab accelerated approval, 6th January 2023. Ten-topic latent Diurich Allocation (LDA) models were created for each corpus either using unigrams or bigrams.</p>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<section id="alzheimers-disease" class="level2">
<h2 class="anchored" data-anchor-id="alzheimers-disease">Alzheimer’s Disease</h2>
<p>Alzheimer’s disease (AD) is a chronic neurodegenerative disease and the most common cause of dementia. Affecting over 55 million people worldwide, the predominant symptoms of AD usually manifest after the age of 65 and include cognitive impairment, as well as physical and emotional difficulties <span class="citation" data-cites="noauthor_2023-ek">(<a href="#ref-noauthor_2023-ek" role="doc-biblioref">Anon, 2023</a>)</span>. These symptoms usually manifest after the physiological changes in the brain develop, therefore scientific understanding and advances in new treatments is important in understanding AD aetiology.</p>
<p>Whilst the underlying mechanisms determining the progression of AD are not fully understood, the accumulation of abnormal protein aggregates including, amyloid-beta plaques and neurofibrillary tangles, are frequently found in the brains of patients with AD <span class="citation" data-cites="Villemagne2013-xk">(<a href="#ref-Villemagne2013-xk" role="doc-biblioref">Villemagne et al., 2013</a>)</span> <span class="citation" data-cites="Iaccarino2018-ta">(<a href="#ref-Iaccarino2018-ta" role="doc-biblioref">Iaccarino et al., 2018</a>)</span>. This leads to disruptions in neuronal signalling pathways and can result in brain atrophy. Diagnosing AD requires the presence of both amyloid and tau pathologies as well as neuroinflammation, neuronal death and brain atrophy <span class="citation" data-cites="Garcia-Morales2021-zb">(<a href="#ref-Garcia-Morales2021-zb" role="doc-biblioref">Garcı́a-Morales et al., 2021</a>)</span>.</p>
</section>
<section id="treatments-for-ad" class="level2">
<h2 class="anchored" data-anchor-id="treatments-for-ad">Treatments for AD</h2>
<p>Although there are currently no therapies or interventions that can cure AD, several disease-modifying treatments exist that can alter the course of the disease, alleviate symptoms, and enhance the overall quality of life for patients. A recent comprehensive review by <span class="citation" data-cites="Huang2023-vq">(<a href="#ref-Huang2023-vq" role="doc-biblioref">Huang et al., 2023</a>)</span>, identified a shift in clinical trial research, which highlighted more Phase I studies being conducted and more Phase III trials involving anti-amyloid therapies. These trials are involving more patients with early onset AD and mild cognitive impairment (MCI) to help develop preventative therapies. Global estimates for people living with preclinical AD or positive for AD pathology biomarkers were 69 and 315 million, respectively <span class="citation" data-cites="Gustavsson2023-qr">(<a href="#ref-Gustavsson2023-qr" role="doc-biblioref">Gustavsson et al., 2023</a>)</span>, therefore increasing research focus on these patient populations is imperative to slow the progression of the disease.</p>
<p>There have recently been two drugs granted approval by the United States Food and Drug Administration (US FDA) which target the pathophysiologies of AD; aducanumab and lecanemab <span class="citation" data-cites="Center_for_Drug_Evaluation2023-gy">(<a href="#ref-Center_for_Drug_Evaluation2023-gy" role="doc-biblioref">Center for Drug Evaluation and Research, 2023</a>)</span>; <span class="citation" data-cites="Office_of_the_Commissioner2023-hu">(<a href="#ref-Office_of_the_Commissioner2023-hu" role="doc-biblioref">Office of the Commissioner, 2023</a>)</span>. These treatments are human monoclonal immunotherapies which aim to target and reduce the beta-amyloid protein aggregates in the brain by binding to its various forms in the amyloid-beta pathway. Furthermore, there are four more anti-amyloid monoclonal antibody treatments which have undergone or are currently in Phase III clinical trials <span class="citation" data-cites="Cummings2023-lo">(<a href="#ref-Cummings2023-lo" role="doc-biblioref">Cummings et al., 2023</a>)</span>. For this study, we will focus on lecanemab, the most recent AD treatment to undergo accelerated approval by the FDA, which was fully approved for treatment in early AD on the 6th July 2023.</p>
</section>
<section id="topic-modelling-and-text-mining-in-ad-research" class="level2">
<h2 class="anchored" data-anchor-id="topic-modelling-and-text-mining-in-ad-research">Topic Modelling and Text Mining in AD Research</h2>
<p>Given the consistent publication of thousands of articles every year concerning AD, focusing on early-stage drug discovery could lead to more research literature and clinical findings being published. Systematic reviews and meta-analyses are time consuming and labour intensive, and pose a significant challenge to updating the current understandings in the research field <span class="citation" data-cites="Higgins2019-kn">(<a href="#ref-Higgins2019-kn" role="doc-biblioref">Higgins, 2019</a>)</span>. Topic modelling, a prominent text mining technique, can find patterns and relationships within natural language data, and could provide an automated and unbiased overview of research text. The most common topic modelling method is Latent Dirichlet Allocation (LDA) which assumes, for unstructured text data like research publication, that each document is made up of a number of topics and that each topic is made up of a collection of words <span class="citation" data-cites="Blei2003-lh">(<a href="#ref-Blei2003-lh" role="doc-biblioref">Blei, Ng and Jordan, 2003</a>)</span>. Each LDA topic is represented as a probability of words within a topic and a probability of topics within each document, which each follow a Dirichlet distribution.</p>
<p>In silico topic modelling has been used for various applications relating to AD, including describing the research landscape. <span class="citation" data-cites="Martinelli2022-ic">(<a href="#ref-Martinelli2022-ic" role="doc-biblioref">Martinelli, 2022</a>)</span>, identifying novel biomarkers <span class="citation" data-cites="Greco2012-pv">(<a href="#ref-Greco2012-pv" role="doc-biblioref">Greco et al., 2012</a>)</span>, and drug repurposing <span class="citation" data-cites="Nian2022-xw">(<a href="#ref-Nian2022-xw" role="doc-biblioref">Nian et al., 2022</a>)</span>. Martinelli performed a nine-topic LDA model and identified five mechanistic themes, one topic relating to AD diagnosis and three concerning treatments. To the best of my knowledge, no studies have explored the change to the AD research landscape with the emergence of newly approved immunotherapy treatments. To increase the findability and reduce bias when selecting articles, the litsearchr package will help identify the most important terms which we will reference our search terms against <span class="citation" data-cites="Grames2019-as">(<a href="#ref-Grames2019-as" role="doc-biblioref">Grames et al., 2019</a>)</span>.</p>
</section>
<section id="aims-and-hypotheses" class="level2">
<h2 class="anchored" data-anchor-id="aims-and-hypotheses">Aims and Hypotheses</h2>
<p>In this study we aim to comprehensively investigate current research in AD, focusing on the period around the accelerated approval of the AD immunotherapy drug, lecanemab. We hypothesised that major topics in AD research could be categorised into distinct themes by applying LDA topic models to existing literature. Additionally, we hope to determine whether these topics have changed with the emergence of new immunotherapy treatments. This method is justified for the vast quantity of literature being published concerning AD and could be translated into other disease areas to study the impact of novel treatment options. Remaining current with the latest research is crucial for making significant strides in understanding and addressing the complexities of this disease, and finding new ways to treat AD.</p>
</section>
</section>
<section id="methods" class="level1">
<h1>Methods</h1>
<p>A full summary of the methodology is provided in Appendix 1. All data analysis and visualisations were done in R version 4.3.2 using <em>tidyverse</em> packages <span class="citation" data-cites="Wickham2019-rj">(<a href="#ref-Wickham2019-rj" role="doc-biblioref">Wickham et al., 2019</a>)</span> unless otherwise stated.</p>
<section id="data-acquisition" class="level2">
<h2 class="anchored" data-anchor-id="data-acquisition">Data Acquisition</h2>
<p>Due to accessing constraints, abstracts represent the only document content for this study. Titles, full abstract text, author list, and publication date were obtained from PubMed using the inclusion criteria described in <a href="#tbl-inclusion-criteria">Table&nbsp;1</a> and <em>EUTils</em>, accessed through <em>Rismed</em> <span class="citation" data-cites="Kovalchik2021-xq">(<a href="#ref-Kovalchik2021-xq" role="doc-biblioref">Kovalchik, 2021</a>)</span>. After excluding publications without abstracts (n = 44), 6155 manuscripts comprised the final dataset (data cut accessed on 10th October 2023). PubMed is a large, online search engine containing over 36 million citations for biomedical literature, and is maintained by the United States National Library of Medicine (NLM) at the National Institutes of Health (NIH). The diversity of entries into the database ensures that the contents are representative and studies are reliable as they are obtained from multiple sources. Entries are assigned Medical Subject Headings (MeSH) which identify health-related terms within each document, therefore classifying articles according to their subject nature which reduces potential interpretive bias.</p>
<div class="cell" data-tbl-width="10" data-tbl-height="3.5" data-tbl-align="left" data-hash="report_cache/html/tbl-inclusion-criteria_d862576a8fa87592d60e8a9cdeecef21">
<div class="cell-output-display">
<div id="tbl-inclusion-criteria" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;1: Inclusion Criteria</caption>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>Criteria:</strong></td>
<td style="text-align: left;"><strong>Filter applied:</strong></td>
</tr>
<tr class="even">
<td style="text-align: left;">MeSH term</td>
<td style="text-align: left;">‘Alzheimer’s Disease’</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Article Type</td>
<td style="text-align: left;">Books</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Case Reports</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Clinical Study</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Clinical Trial</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Controlled Clinical Trial</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Meta-analysis</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Randomised Controlled Trial</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Review</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Systematic Review</td>
</tr>
<tr class="even">
<td style="text-align: left;">Publication Date</td>
<td style="text-align: left;">1st January 2022 to 1st January 2024 inclusive</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Language</td>
<td style="text-align: left;">English</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</section>
<section id="data-preprocessing" class="level2">
<h2 class="anchored" data-anchor-id="data-preprocessing">Data Preprocessing</h2>
<p>Abstracts and their metadata were categorised into “pre-leca” or “post-leca” corpuses based on their publication date relative to the date of lecanemab’s accelerated early approval, 6th January 2023. Full abstract text was tokenised into single words using the <em>unnest_tokens</em> function of the <em>tidytext</em> package <span class="citation" data-cites="Silge2016-jf">(<a href="#ref-Silge2016-jf" role="doc-biblioref">Silge and Robinson, 2016</a>)</span>. The same function was used for tokenising to bigrams and trigrams, using n = 2 and n = 3 respectively. Stop words from the <em>tidytext</em> package <span class="citation" data-cites="Silge2016-jf">(<a href="#ref-Silge2016-jf" role="doc-biblioref">Silge and Robinson, 2016</a>)</span> combined with personalised words (<a href="#tbl-appendix2">Table&nbsp;2</a>) were then removed <a href="#fig-preprocess">Figure&nbsp;1</a>. To prevent the different spellings of the same word or phrase from being counted multiple times, similar words were mapped to the same variable. For example, ‘<em>amyloid β</em>, ‘<em>beta amyloid</em>’, and ‘<em>amyloid aβ</em>’ were all mapped to ‘<em>amyloid beta</em>’.</p>
<div id="fig-preprocess" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-preprocess-1" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="data/preprocess1.png" class="img-fluid figure-img" data-ref-parent="fig-preprocess" width="480"></p>
<figcaption class="figure-caption">(a) Example of Tokenisation</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-preprocess-2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="data/preprocess2.png" class="img-fluid figure-img" data-ref-parent="fig-preprocess" width="480"></p>
<figcaption class="figure-caption">(b) Removal of stopwords</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;1: <strong>Abstract tokenisation into unigrams.</strong> Schematic of an abstract being (A) tokenised into single-word tokens followed by (B) removal of stop words obtained from the <em>tidytext</em> package <span class="citation" data-cites="Silge2016-jf">(<a href="#ref-Silge2016-jf" role="doc-biblioref">Silge and Robinson, 2016</a>)</span>, as well as personalised words (<a href="#tbl-appendix2">Table&nbsp;2</a>). Tokenisation and data cleaning of bigrams and trigrams followed the same methods, not shown.</figcaption>
</figure>
</div>
</section>
<section id="data-analysis" class="level2">
<h2 class="anchored" data-anchor-id="data-analysis">Data Analysis</h2>
<section id="litsearchr" class="level3">
<h3 class="anchored" data-anchor-id="litsearchr"><em>litsearchR</em></h3>
<p>To reassure us that the PubMed search query encapsulated all literature, <em>litsearchr</em> package was used to expand the search terms <span class="citation" data-cites="Grames2019-as">(<a href="#ref-Grames2019-as" role="doc-biblioref">Grames et al., 2019</a>)</span>. Citations from PubMed results using the previous search criteria were read into R. The combined unique keyword and titles, as not all articles have keywords, for each result were collected. To ensure only the most relevant terms were searched, stop words were removed, as previously described, and the minimum frequency of words was set as n = 50 for keywords and n = 75 for the title. A matrix of each word in each article was created and the potential search terms were ranked with <em>create_network</em> and <em>strength</em> <span class="citation" data-cites="Barrat2004-hm">(<a href="#ref-Barrat2004-hm" role="doc-biblioref">Barrat et al., 2004</a>)</span> from the <em>igraph</em> package <span class="citation" data-cites="Csardi2006-gb">(<a href="#ref-Csardi2006-gb" role="doc-biblioref">Csardi and Nepusz, 2006</a>)</span>. The 80% cutoffs were determined to find only the most important terms to the articles; all others were discarded (<a href="#fig-ad-search-terms">Figure&nbsp;2</a>; <a href="#fig-ad-drug-search-terms">Figure&nbsp;3</a>).</p>
<div class="cell" data-layout-align="left" data-hash="report_cache/html/fig-ad-search-terms_5233d7133121e551742dd0a2fd054161">
<div class="cell-output-display">
<div id="fig-ad-search-terms" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-ad-search-terms-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption class="figure-caption">Figure&nbsp;2: Search term network for Alzheimer’s Disease</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell" data-layout-align="left" data-hash="report_cache/html/fig-ad-drug-search-terms_5d1e45dfc7b94a930f07ec61b9263538">
<div class="cell-output-display">
<div id="fig-ad-drug-search-terms" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-ad-drug-search-terms-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption class="figure-caption">Figure&nbsp;3: AD drug search term network</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="publication-frequency" class="level3">
<h3 class="anchored" data-anchor-id="publication-frequency">Publication Frequency</h3>
<p>Number of abstracts published per month were visualised as well as the frequency of dates of publications for papers containing the AD drug lecanemab and associated terminology were also obtained.</p>
</section>
<section id="term-frequency" class="level3">
<h3 class="anchored" data-anchor-id="term-frequency">Term Frequency</h3>
<section id="n-gram-frequency-analysis" class="level4">
<h4 class="anchored" data-anchor-id="n-gram-frequency-analysis">N-gram Frequency Analysis</h4>
<p>After tokenisation, the top 20 most frequent unigrams were determined for each dataset. The top 20 most frequent bigrams and trigrams were also determined due to many unigrams being associated with pairs or triplets of words. For example, “mild cognitive impairment” relates to a neurological condition, whereas the words “mild”, “cognitive” and “impairment” have ambiguous connotations individually. The most frequent bigrams were also visualised as networks.</p>
</section>
<section id="term-usage-over-time" class="level4">
<h4 class="anchored" data-anchor-id="term-usage-over-time">Term Usage Over Time</h4>
<p>The distribution of terms used over 1000 times from the unigram analysis was visualised. Generalised linear model (GLM) was used to determine whether there was a significant change in word usage over the months.</p>
</section>
</section>
<section id="topic-modelling" class="level3">
<h3 class="anchored" data-anchor-id="topic-modelling">Topic Modelling</h3>
<p>A document term matrix (dtm) was constructed for each dataset, indicating each word’s term frequency (tf), which is a measure of how often a word appears in a document. To determine if a statistical model could distinguish between pre-leca and post-leca corpuses, a two-topic Latent Dirich Allocation (LDA) model <span class="citation" data-cites="Blei2003-lh">(<a href="#ref-Blei2003-lh" role="doc-biblioref">Blei, Ng and Jordan, 2003</a>)</span> was applied to the dtm using the topicmodels package <span class="citation" data-cites="Grun2011-do">(<a href="#ref-Grun2011-do" role="doc-biblioref">Grün and Hornik, 2011</a>)</span>. The per-document-per-topic probabilities (γ) is extracted to show the proportion of words generated in each topic and how often these words appear in either the pre-leca or post-leca corpuses.</p>
<p>Furthermore, two ten-topic LDA models were created, one for each of the pre-leca and post-leca text corpuses, to determine the most frequent topics, where an arbitrary topic number (k) of ten was chosen. The per-topic-per-word probabilities (β) were extracted and the top 10 terms most commonly found in each topic were visualised. As a lot of the most common words appear as bigrams in the text corpuses, the per-topic-per-bigram probabilities with the top 10 most common bigrams were also visualised with two more ten-topic LDA models. In each model the abstracts are considered mixtures of topics and each topic is considered a mixture of words.</p>
</section>
</section>
</section>
<section id="results-and-discussion" class="level1">
<h1>Results and Discussion</h1>
<section id="ad-research-publication-are-similar-over-time" class="level2">
<h2 class="anchored" data-anchor-id="ad-research-publication-are-similar-over-time">AD Research Publication are Similar Over Time</h2>
<p>This study found 6155 papers that were published between 2022-01-01 and 2023-10-10 that contained the identified terms (<strong>?@tbl-inclusion</strong> criteria).</p>
<p>The distribution of publication dates for these articles is shown in <a href="#fig-publication-date">Figure&nbsp;4</a>. There were 207 papers published between 01-03-2017 and 31-10-2023 that contained the identified terms.</p>
<div id="fig-publication-date" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-publication-date-1" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-publication-date-1.png" class="img-fluid figure-img" data-ref-parent="fig-publication-date" width="960"></p>
<figcaption class="figure-caption">(a) All Abstracts</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-publication-date-2" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-publication-date-2.png" class="img-fluid figure-img" data-ref-parent="fig-publication-date" width="960"></p>
<figcaption class="figure-caption">(b) Lecanemab Abstracts</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;4: <strong>Frequency of lecanemab publications increases in 2023.</strong> Distribution of articles published over time containing (A) the MeSH term ‘Alzheimer’s Disease’ in the title and/or abstract (n = 6,155) or (B) terms associated with the AD drug lecanemab: ‘lecanemab’, ‘leqembi’, ‘BAN2401’, and ‘mAb158’ (n = 207). Terms associated with lecanemab are also in black in (A). Line represents date of lecanemab accelerated approval.</figcaption>
</figure>
</div>
<p>To determine if a statistical model could distinguish between pre-leca and post-leca corpuses, a two-topic Latent Dirich Allocation (LDA) model <span class="citation" data-cites="Blei2003-lh">(<a href="#ref-Blei2003-lh" role="doc-biblioref">Blei, Ng and Jordan, 2003</a>)</span> was applied to the dtm using the <em>topicmodels</em> package <span class="citation" data-cites="Grun2011-do">(<a href="#ref-Grun2011-do" role="doc-biblioref">Grün and Hornik, 2011</a>)</span>. The per-document-per-topic probabilities (<em>γ</em>) is extracted to show the proportion of words generated in each topic and how often these words appear in either the pre-leca or post-leca corpuses.</p>
<div class="cell" data-layout-align="left" data-hash="report_cache/html/fig-gamma-lda_ff2ec9f23549e3f45499d1446d8bc4b6">
<div class="cell-output-display">
<div id="fig-gamma-lda" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-gamma-lda-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption class="figure-caption">Figure&nbsp;5: <strong>Two topics show similar occurrences around lecanemab approval.</strong> No differences were observed between pre- and post-leca corpuses in a two topic model showing the per-document-per-topic probabilities, γ. Lines represent mean γ scores and points represent individual γ scores, n = 6,155. Abstracts split into two topics then significance shows if each topic is more related to one of the pre- or post-leca corpuses. Two topics show similar occurrences around lecanemab approval. Two topic model showing the per-document-per-topic probabilities, γ for full abstract dataset (n = 6,155), then grouped by pre-leca or post-leca text corpuses. Lines represent mean γ scores and points represent individual γ scores.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="n-gram-analysis" class="level2">
<h2 class="anchored" data-anchor-id="n-gram-analysis">N-gram Analysis</h2>
<p>Despite the full abstract dataset not significantly splitting into two distinct topics, we aimed to suggest this was because the language was very similar between the two corpses. We therefore aimed to explore the most common n-grams frequencies. The most frequent unigrams, bigrams and trigrams are shown in <a href="#fig-tokenisation">Figure&nbsp;6</a>. ‘<em>Disease</em>’ is the most frequent unigram in both corpuses due to being most frequently associated with ‘<em>neurodegenerative disease</em>’ as well as specifically for ‘<em>Parkinson’s disease</em>’, and ‘<em>Huntington’s disease</em>’ which are common neurodegenerative diseases which are researched as much as AD.</p>
<div id="fig-tokenisation" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-tokenisation-1" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-tokenisation-1.png" class="img-fluid figure-img" data-ref-parent="fig-tokenisation" width="960"></p>
<figcaption class="figure-caption">(a) Unigram</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-tokenisation-2" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-tokenisation-2.png" class="img-fluid figure-img" data-ref-parent="fig-tokenisation" width="960"></p>
<figcaption class="figure-caption">(b) Bigram</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-tokenisation-3" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-tokenisation-3.png" class="img-fluid figure-img" data-ref-parent="fig-tokenisation" width="960"></p>
<figcaption class="figure-caption">(c) Trigram</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;6: <strong>Top 11 most frequent unigrams in AD literature are the same before and after lecanemab approval.</strong> Tokenised unigrams split into pre- and post-lecanemab corpuses and ranked by (A) unigram frequency and (B) bigram frequency removing personalised stop-words (<a href="#tbl-appendix2">Table&nbsp;2</a>) (n = 6155).</figcaption>
</figure>
</div>
<p>The distribution by month of the top 10 frequent words, as determined from <a href="#fig-tokenisation">Figure&nbsp;6</a>, is shown in <a href="#fig-glm-words">Figure&nbsp;7</a>.</p>
<div class="cell" data-layout-align="left" data-hash="report_cache/html/fig-glm-words_7adfbf27f33a8147b8a8e916ef2c51b2">
<div class="cell-output-display">
<div id="fig-glm-words" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-glm-words-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption class="figure-caption">Figure&nbsp;7: <strong>Word frequency trends are similar for the top 10 most frequent terms.</strong> The focus of AD research has stayed consistent over time. The frequency of a selection of words used over 1000 times in AD abstracts from 01-01-2022 to 10-10-2023 has not changed.</figcaption>
</figure>
</div>
</div>
</div>
<p>As ‘<em>disease</em>’ was the most common unigram, we explored the most frequent bigrams that end in ‘<em>disease</em>’ (<a href="#fig-bigram-plot">Figure&nbsp;8</a>). We also explored the most common bigrams that begin with ‘<em>neuro</em>’ and those that end in ‘<em>neuro</em>’ to understand the context in which these words are used.</p>
<div id="fig-bigram-plot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div id="fig-bigram-plot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">

<div class="wordcloud2 html-widget html-fill-item" id="htmlwidget-1b4ff99564eb6e8884a5" style="width:100%;height:433px;"></div>
<script type="application/json" data-for="htmlwidget-1b4ff99564eb6e8884a5">{"x":{"word":["parkinson's","neurodegenerative","huntington's","alzheimer","cardiovascular","parkinson","vessel","coronavirus","neurological","cerebrovascular","liver","heart","bowel","multifactorial","periodontal","human","brain","huntington","effective","related","chronic","degenerative","kidney","metabolic","prion","pulmonary","body","potential","jakob","underlying","neuron","vascular","artery","clinical","pick's","alzheimer’s","neurologic","renal","complex","crohn's","devastating","grain","inflammatory","progressive","specific","autoimmune","cns","severe","slow","understanding","debilitating","earlier","future","gaucher","lung","systemic","wilson's","based","cardiac","cardiological","common","delay","developing","hakola","halt","infectious","modify","psychiatric","system","identify","multiple","potentially","prevent","relevant","tooth","advanced","ameliorate","celiac","develop","improve","influence","matter","pd","promising","reducing","respiratory","skin","stage","understand","19","addison's","affect","alzheimers","alzheimeŕs","cell","cerebral","challenging","dementia","diagnosis","evaluating","eye","halting","heterogeneous","identifying","include","incurable","late","mitochondrial","model","monitor","monitoring","muscle","neuronal","neuropathological","neuropsychiatric","preclinical","predicting","provide","rare","reduce","slowing","sporadic","stop","study","1","accelerate","accurate","age","aging","alleviate","alter","assess","batten","causing","cognitive","crohn","cross","current","deadly","decipher","define","delayed","demyelinating","determine","diagnose","diverse","emerging","existing","facilitate","fnirs","gaucher's","global","growing","hypothetical","inclusion","increased","investigating","ischemic","longitudinal","lyme","mediated","menkes","microvascular","mitigating","modifies","molecular","multifaceted","neuroinflammatory","nyhan","ovarian","overlapping","paget's","parkinson’s","pick","predict","predominant","preventing","promote","promoting","putative","reflecting","reflux","reverse","scheinker","significant","single","slowed","subtle","tackle","targeted","treat","tropical","ulcer","wasting","wilson","14","161","39","64","77.76","8","abating","ablate","absent","accelerated","accelerates","accessible","accomplish","achieving","active","address","admission","advance","advancing","affects","aggressive","allergic","alleviating","alterations","altered","altering","alternative","alzheimer.s","analysis","analyze","anatomy","ancl","animal","anticipating","approved","arrest","assessing","asymptomatic","atherosclerotic","attenuate","atypical","autoimmunity","aß","basic","bayesian","binswanger","biological","biospecimens","bipolar","blinding","blocking","booster","borna","buffer","bullous","candidate","cardiometabolic","care","carotid","cataract","causative","cerebellar","change","characteristics","characterizing","childhood","clinicopathologic","cm","collaborative","combat","combined","comedications","communicable","complete","complicate","complicated","concordant","considered","contagious","control","controlling","controls","convergent","core","coronary","cow","create","curable","da1","da2","damaging","databases","decades","decelerates","defer","definitive","delaying","delineate","dementing","demonstrate","demonstrated","demyelinated","dental","depression","detecting","development","diagnosed","diagnostic","differential","directed","directing","disabling","discover","discovery","disease","diseases","disorders","distinct","distinguish","diversified","dominant","dot","drives","drug","dull","dynamic","dysfunctions","e.g","earliest","elucidate","elucidating","endpoints","enhance","established","estimating","etiology","evaluate","exacerbate","exacerbating","exerts","expediate","experimental","extensive","faceted","factor","factors","faster","features","finding","fluctuating","forestalling","friedreich's","gastrointestinal","geriatric","glioma","globally","guiding","hallmark","halts","hamper","hansen's","hd","health","hepatic","hintington's","hirschsprung","hirschsprung's","host","hypertension","i.e","ideal","identified","immunomodulatory","impact","impacts","impede","impending","implant","implementing","improved","infection","inflict","inhibiting","initial","initiate","innovative","inter","intermediate","intra","introduced","invasive","jakob's","joseph","key","lack","lacking","lafora","life","limited","lundborg","macular","major","manifesting","mci","meaningful","mediating","menière's","merzbacher","methodology","microbiome","milder","misfolding","mitigate","modeling","moderate","modernized","modest","modifying","modulate","modulates","modulating","molecules","mouse","natural","neoplastic","neural","neurogenerative","neuroinfectious","neuroinflammation","neuromuscular","neurone","numerous","observed","obtain","occlusive","ocular","offending","ongoing","optimizing","oral","organoid","oriented","original","outcomes","paget","parkinsons","parkinsons's","parknson's","participants","pathophysiological","perplexing","phenotype","picks","pleural","polyq","pompe","possibly","precise","prevalent","pro","prodromal","projected","proteinopathic","proteins","psychosomatic","pure","ra","rapid","recapitulate","recapitulates","receive","recently","relapsed","relevance","reliable","relieve","report","reported","respective","response","reveal","revealed","rodent","schizophrenia","seeding","severity","sex","signaling","silent","slows","sole","spine","storage","stratifying","stress","studied","studying","subclinical","subjective","subsequent","substantial","suggested","superior","supporting","symptomatic","symptoms","tackles","targeting","tauopathy","terrible","therapeutic","threatening","throat","tissue","tracking","treating","trials","triggers","typical","ultimately","uncover","underlie","unifying","unique","unpredictable","unravel","urine","vcp","viable","viral","vitro","wm","worldwide","worse","worsen","xiaoke","zebrafish"],"freq":[1393,544,325,309,131,59,55,49,48,43,39,38,37,32,32,30,26,25,23,23,21,21,21,20,20,20,17,17,16,15,14,14,13,13,13,11,11,11,10,10,10,10,10,10,10,9,9,9,9,8,7,7,7,7,7,7,7,6,6,6,6,6,6,6,6,6,6,6,6,5,5,5,5,5,5,4,4,4,4,4,4,4,4,4,4,4,4,4,4,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],"fontFamily":"Segoe UI","fontWeight":"bold","color":"random-dark","minSize":0,"weightFactor":0.1292175161521895,"backgroundColor":"white","gridSize":0,"minRotation":-0.7853981633974483,"maxRotation":0.7853981633974483,"shuffle":true,"rotateRatio":0.4,"shape":"circle","ellipticity":0.65,"figBase64":null,"hover":null},"evals":[],"jsHooks":{"render":[{"code":"function(el,x){\n                        console.log(123);\n                        if(!iii){\n                          window.location.reload();\n                          iii = False;\n\n                        }\n  }","data":null}]}}</script>
<figcaption class="figure-caption">(a) Most common bigrams ending in ‘disease’</figcaption>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;8: <em>Bigrams exploration</em></figcaption>
</figure>
</div>
<section id="bigram-networks" class="level3">
<h3 class="anchored" data-anchor-id="bigram-networks">Bigram Networks</h3>
<p>Relations of bigram networks are shown in <a href="#fig-bigram-visualisation">Figure&nbsp;9</a>.</p>
<div id="fig-bigram-visualisation" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-bigram-visualisation-1" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-bigram-visualisation-1.png" class="img-fluid figure-img" data-ref-parent="fig-bigram-visualisation" width="960"></p>
<figcaption class="figure-caption">(a) All Bigram Relations</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-bigram-visualisation-2" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-bigram-visualisation-2.png" class="img-fluid figure-img" data-ref-parent="fig-bigram-visualisation" width="960"></p>
<figcaption class="figure-caption">(b) Pre-leca Bigram Relations</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-bigram-visualisation-3" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-bigram-visualisation-3.png" class="img-fluid figure-img" data-ref-parent="fig-bigram-visualisation" width="960"></p>
<figcaption class="figure-caption">(c) Post-leca Bigram Relations</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;9: <strong>‘Disease’ commonly occurs with ‘neurodegenerative’ and ‘parkinson’s’ in bigram relationship networks.</strong> (A) All abstracts. (B) Pre-leca bigram corpus. (C) Post-leca bigram corpus. Increased thickness indicates increased frequency.</figcaption>
</figure>
</div>
</section>
</section>
<section id="topic-modelling-1" class="level2">
<h2 class="anchored" data-anchor-id="topic-modelling-1">Topic Modelling</h2>
<section id="unigrams" class="level3">
<h3 class="anchored" data-anchor-id="unigrams">Unigrams</h3>
<p>The top 10 unigrams in LDA for <strong>pre-leca:</strong> and <strong>post-leca</strong> are shown in <a href="#fig-topic-model-unigrams">Figure&nbsp;10</a>.</p>
<ul>
<li><p><strong>Topic 1:</strong> Clinical studies - ‘treatment’, ‘trials’, ‘development’, ‘clinical’</p></li>
<li><p><strong>Topic 2:</strong> Studies for diagnosis - ‘clinical’, ‘data’, ‘biomarkers’, ‘diagnosis’</p></li>
<li><p><strong>Topic 3:</strong> Risk factors &amp; symptoms - ‘age’, ‘health’, ‘depression’</p></li>
<li><p><strong>Topic 4:</strong> Cells - ‘neuroinflammation’, ‘microglia’, ‘signaling’</p></li>
<li><p><strong>Topic 5:</strong> ? - ‘sleep’, ‘inflammatory’, ‘effects’</p></li>
<li><p><strong>Topic 6:</strong> Mechanisms ? - ‘mitochondrial’, ‘gut’, ‘stress’, ‘brain’</p></li>
<li><p><strong>Topic 7:</strong> Genetics - ‘gene’, ‘risk’, ‘common’</p></li>
<li><p><strong>Topic 8:</strong> Abnormal proteins - ‘tau’, ‘aB’, ‘plaques’, ‘aggregation’, ‘accumulation’, ‘beta’</p></li>
<li><p><strong>Topic 9:</strong> Study criteria ? - ‘mci’, ‘included’, ‘meta’, ‘cognitive’ - i.e.&nbsp;choosing the correct people for a study is very important to ensure you get the correct findings, especially choosing AD patients with mild vs late MCI vs clinical dementia &amp; AD</p></li>
<li><p><strong>Topic 10:</strong> Brain - ‘neurological’, ‘neurodegeneration’, ‘blood brain barrier’, ‘system’</p></li>
</ul>
<p>The top 10 unigrams in LDA for <strong>post-leca:</strong></p>
<ul>
<li><p>Topic 1: Study criteria - ‘mci’, ‘patients’, ‘included’, ‘dementia’</p></li>
<li><p>Topic 2: Brain - ‘memory’, ‘disorders’, ‘imaging’, ‘neurodegeneration’</p></li>
<li><p>Topic 3: ? - ‘covid’, ‘19’, ‘tau’, ‘protein’</p></li>
<li><p>Topic 4:</p></li>
<li><p>Topic 5: Risk Factors - ‘sleep’, ‘health’, ‘age’, ‘ageing’</p></li>
<li><p>Topic 6: Treatments - ‘therapeutics’, ‘effects’, ‘anti’, ‘drugs’, ‘treatment’</p></li>
<li><p>Topic 7: Study ? - ‘study’, ‘participants’, ‘clinical’, ‘risk’</p></li>
<li><p>Topic 8: Diagnosis - ‘biomarkers’, ‘genetic’, ‘research’, ‘analysis’</p></li>
<li><p>Topic 9: Proteins - ‘amyloid’, ‘aB’, ‘tau’, ‘brain’, ‘B’, ‘protein’</p></li>
<li><p>Topic 10: Diabetes ? - ‘insulin’, ‘receptor’, ‘diabetes’, ‘1’, ‘2’ - though 1 might be from something else, e.g.&nbsp;‘1 42’ is a bigram which is a type of amyloid beta peptide 1-40 and 1-42</p></li>
</ul>
<div id="fig-topic-model-unigrams" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-topic-model-unigrams-1" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="plots/pre-leca-lda-updated.png" class="img-fluid figure-img" data-ref-parent="fig-topic-model-unigrams" width="1012"></p>
<figcaption class="figure-caption">(a) Pre-leca</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-topic-model-unigrams-2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="plots/post-leca-lda-updated.png" class="img-fluid figure-img" data-ref-parent="fig-topic-model-unigrams" width="1012"></p>
<figcaption class="figure-caption">(b) Post-leca</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;10: <strong>Unigram LDA Topic Modelling.</strong> Outputs from the model with ten topics. Figures show the top ten most commonly associated unigrams in each topic ordered by their per-topic-per-word probability, β. Topic titles were manually created and added.</figcaption>
</figure>
</div>
</section>
<section id="bigrams" class="level3">
<h3 class="anchored" data-anchor-id="bigrams">Bigrams</h3>
<p>The top 10 bigrams in topic models for pre- and post- leca corpuses are shown in <a href="#fig-topic-model-bigrams">Figure&nbsp;11</a>. We removed the bigrams ‘<em>neurodegenerative disease</em>’ and ‘<em>parkinson’s disease</em>’ from the models as they were present in all topics. Additionally we removed ‘<em>amyloid-beta</em>’ from the pre-leca model as this was also present in all topics.</p>
<p>Then most frequent terms:</p>
<ul>
<li><p>‘<strong>cognitive impairment</strong>’ was common to 8/10 pre-leca models and 9/10 post-leca</p></li>
<li><p>‘<strong>amyloid beta</strong>’ was common to 8/10 post-leca models</p></li>
</ul>
<p>Unique terms which define each topic:</p>
<p>Themes of topics <strong>pre-leca</strong>:</p>
<ul>
<li><p><strong>Topic 1:</strong> Risk factors ? - ‘diabetes t2d’, ‘insulin resistance’, ’</p></li>
<li><p><strong>Topic 2:</strong></p></li>
<li><p><strong>Topic 3:</strong></p></li>
<li><p><strong>Topic 4:</strong> “Covid 19” - high beta value</p></li>
<li><p><strong>Topic 5:</strong></p></li>
<li><p><strong>Topic 6:</strong> “Amyloid beta” - high beta value</p></li>
<li><p><strong>Topic 7:</strong></p></li>
<li><p><strong>Topic 8:</strong></p></li>
<li><p><strong>Topic 9:</strong></p></li>
<li><p><strong>Topic 10:</strong></p></li>
</ul>
<p>Themes of topics <strong>post-leca:</strong></p>
<ul>
<li><p>Topic 1: neurodegenerative diseases - “huntington’s disease”, “ALS”</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 2:</p></li>
</ul>
<div id="fig-topic-model-bigrams" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-topic-model-bigrams-1" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-topic-model-bigrams-1.png" class="img-fluid figure-img" data-ref-parent="fig-topic-model-bigrams" width="1440"></p>
<figcaption class="figure-caption">(a) Pre-leca</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-topic-model-bigrams-2" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-topic-model-bigrams-2.png" class="img-fluid figure-img" data-ref-parent="fig-topic-model-bigrams" width="1440"></p>
<figcaption class="figure-caption">(b) Post-leca</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;11: <strong>Bigram LDA Topic Modelling.</strong> Outputs from the post-lecanemab (after 06/01/2023) corpus model with ten topics. Figures show the top ten most commonly associated bigrams in each topic ordered by their per-topic-per-word probability, β. Topic titles were manually created and added.</figcaption>
</figure>
</div>
</section>
</section>
<section id="trigrams" class="level2">
<h2 class="anchored" data-anchor-id="trigrams">Trigrams</h2>
<p>The top 10 trigrams in topic models for pre- and post- leca corpuses are shown in <a href="#fig-topic-model-trigrams">Figure&nbsp;12</a>. We removed the trigrams ‘<em>central nervous system</em>’ and ‘<em>parkingson’s disease pd</em>’ from the models as they were present in all topics and did not provide any additional information.</p>
<p>Themes of topics <strong>pre-leca</strong>:</p>
<ul>
<li><p>Topic 1:</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 3:</p></li>
<li><p>Topic 4: Covid (?)</p></li>
<li><p>Topic 5:</p></li>
<li><p>Topic 6:</p></li>
<li><p>Topic 2:</p></li>
</ul>
<p>Themes of topics <strong>post-leca:</strong></p>
<ul>
<li><p>Topic 1:</p></li>
<li><p>Topic 2:</p></li>
<li><p>Topic 3: “chinese patent medicine”</p></li>
<li><p>Topic 4: “inflammatory bowel disease”</p></li>
<li><p>Topic 5: “bstructive sleep apnea”</p></li>
<li><p>Topic 6:</p></li>
<li><p>Topic 7: “glp 1 ras”</p></li>
<li><p>Topic 8:</p></li>
<li><p>Topic 9: “traditional chinese medicine”</p></li>
<li><p>Topic 10: “donepezil patch 27.5”, “patch 27.5 mg”</p></li>
</ul>
<div id="fig-topic-model-trigrams" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="cell-output-display">
<div id="fig-topic-model-trigrams-1" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-topic-model-trigrams-1.png" class="img-fluid figure-img" data-ref-parent="fig-topic-model-trigrams" width="1440"></p>
<figcaption class="figure-caption">(a) Pre-leca</figcaption>
</figure>
</div>
</div>
<div class="cell-output-display">
<div id="fig-topic-model-trigrams-2" class="quarto-figure quarto-figure-left anchored">
<figure class="figure">
<p><img src="report_files/figure-html/fig-topic-model-trigrams-2.png" class="img-fluid figure-img" data-ref-parent="fig-topic-model-trigrams" width="1440"></p>
<figcaption class="figure-caption">(b) Post-leca</figcaption>
</figure>
</div>
</div>
<figcaption class="figure-caption">Figure&nbsp;12: <strong>Trigram LDA Topic Modelling.</strong> LDA model shows the top ten most commonly associated trigrams in each topic ordered by their per-topic-per-word probability, β, for (A) pre-leca and (B) post-leca corpuses. Topic titles were manually created and added.</figcaption>
</figure>
</div>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<section id="study-limitations" class="level2">
<h2 class="anchored" data-anchor-id="study-limitations">Study Limitations</h2>
<p>Due to our search strategy, a lot of papers contain the words ‘Alzheimer’s disease’ may have been mentioned as a collective with other neurodegenerative diseases. Therefore, the unigrams ‘<em>huntington’s disease</em>’, ‘<em>parkinson’s disease</em>’ and ‘<em>amylotropic lateral sclerosis</em>’ are very frequent.</p>
<p>One reason for there not being any statistical significance between the per-document-per-topic probabilities could be due to the time frame used for the analysis not being long enough to account for a change in the research landscape.</p>
<p>Unable to mine keywords included in the full text, so terms only described in the abstract, regardless of whether other phrases are included in the complete article.</p>
</section>
<section id="conclusion" class="level2">
<h2 class="anchored" data-anchor-id="conclusion">Conclusion</h2>
<p>The results of this study suggest that the introduction of lecanemab has not had a significant impact on the research landscape of neurodegenerative diseases. The most common topics in the pre-lecanemab corpus were also the most common in the post-lecanemab corpus.</p>
</section>
</section>

<section id="appendix" class="level1">
<h1>Appendix</h1>
<div class="cell" data-tbl-width="10" data-tbl-height="3.5" data-tbl-align="left" data-hash="report_cache/html/tbl-appendix2_89066a4e2a1e041a0769ce95d54e9ee5">
<div class="cell-output-display">
<div id="tbl-appendix2" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;2: Personalised Stop Words</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Stop Word</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">alzheimer’s</td>
</tr>
<tr class="even">
<td style="text-align: left;">ad</td>
</tr>
<tr class="odd">
<td style="text-align: left;">95</td>
</tr>
<tr class="even">
<td style="text-align: left;">ci</td>
</tr>
<tr class="odd">
<td style="text-align: left;">including</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>

</section>

<div id="quarto-appendix" class="default"><section id="acknowledgements" class="level1 appendix"><h2 class="anchored quarto-appendix-heading">Acknowledgements</h2><div class="quarto-appendix-contents">

<p>I would like to thank my supervisor Emma Rand for her constant support and guidance throughout my project. I would also like to dedicate this project to my late granddad Alan Scrimshire who passed away on the 31st March 2023 after fighting a five year battle with Alzheimer’s Disease. I hope this project highlights the complexity of the disease and the vast efforts being undertaken to find a cure.</p>
</div></section><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" role="list">
<div id="ref-noauthor_2023-ek" class="csl-entry" role="listitem">
(2023). <span class="nodecoration">2023 alzheimer’s disease facts and figures</span>. 19 (4), pp.1598–1695.
</div>
<div id="ref-Barrat2004-hm" class="csl-entry" role="listitem">
Barrat, A., Barthélemy, M., Pastor-Satorras, R. and Vespignani, A. (2004). <span class="nodecoration">The architecture of complex weighted networks</span>. <em>Proc. Natl. Acad. Sci. U. S. A.</em>, 101 (11), pp.3747–3752.
</div>
<div id="ref-Blei2003-lh" class="csl-entry" role="listitem">
Blei, D. M., Ng, A. Y. and Jordan, M. I. (2003). <span class="nodecoration"><em>Latent dirichlet allocation</em></span>. <a href="https://www.jmlr.org/papers/volume3/blei03a/blei03a.pdf?ref=https://githubhelp.com" class="uri">https://www.jmlr.org/papers/volume3/blei03a/blei03a.pdf?ref=https://githubhelp.com</a>.
</div>
<div id="ref-Center_for_Drug_Evaluation2023-gy" class="csl-entry" role="listitem">
Center for Drug Evaluation and Research. (2023). <span class="nodecoration"><em><span class="nocase">FDA’s</span> decision to approve new treatment for alzheimer’s disease</em></span>. <a href="https://www.fda.gov/drugs/our-perspective/fdas-decision-approve-new-treatment-alzheimers-disease" class="uri">https://www.fda.gov/drugs/our-perspective/fdas-decision-approve-new-treatment-alzheimers-disease</a>; FDA.
</div>
<div id="ref-Csardi2006-gb" class="csl-entry" role="listitem">
Csardi, G. and Nepusz, T. (2006). <span class="nodecoration"><em>The igraph software package for complex network research</em></span>. Complex Systems, p.1695.
</div>
<div id="ref-Cummings2023-lo" class="csl-entry" role="listitem">
Cummings, J., Zhou, Y., Lee, G., Zhong, K., Fonseca, J. and Cheng, F. (2023). <span class="nodecoration">Alzheimer’s disease drug development pipeline: 2023</span>. <em>Alzheimers. Dement.</em>, 9 (2), p.e12385.
</div>
<div id="ref-Garcia-Morales2021-zb" class="csl-entry" role="listitem">
Garcı́a-Morales, V., González-Acedo, A., Melguizo-Rodrı́guez, L., Pardo-Moreno, T., Costela-Ruiz, V. J., Montiel-Troya, M. and Ramos-Rodrı́guez, J. J. (2021). <span class="nodecoration">Current understanding of the physiopathology, diagnosis and therapeutic approach to alzheimer’s disease</span>. <em>Biomedicines</em>, 9 (12).
</div>
<div id="ref-Grames2019-as" class="csl-entry" role="listitem">
Grames, E. M., Stillman, A. N., Tingley, M. W. and Elphick, C. S. (2019). <span class="nodecoration">An automated approach to identifying search terms for systematic reviews using keyword co‐occurrence networks</span>. <em>Methods Ecol. Evol.</em>, 10 (10), Wiley., pp.1645–1654.
</div>
<div id="ref-Greco2012-pv" class="csl-entry" role="listitem">
Greco, I., Day, N., Riddoch-Contreras, J., Reed, J., Soininen, H., Kłoszewska, I., Tsolaki, M., Vellas, B., Spenger, C., Mecocci, P., Wahlund, L.-O., Simmons, A., Barnes, J. and Lovestone, S. (2012). <span class="nodecoration">Alzheimer’s disease biomarker discovery using in silico literature mining and clinical validation</span>. <em>J. Transl. Med.</em>, 10, p.217.
</div>
<div id="ref-Grun2011-do" class="csl-entry" role="listitem">
Grün, B. and Hornik, K. (2011). <span class="nodecoration">Topicmodels: An <span>R</span> package for fitting topic models</span>. <em>J. Stat. Softw.</em>, 40, pp.1–30.
</div>
<div id="ref-Gustavsson2023-qr" class="csl-entry" role="listitem">
Gustavsson, A., Norton, N., Fast, T., Frölich, L., Georges, J., Holzapfel, D., Kirabali, T., Krolak-Salmon, P., Rossini, P. M., Ferretti, M. T., Lanman, L., Chadha, A. S. and Flier, W. M. van der. (2023). <span class="nodecoration">Global estimates on the number of persons across the alzheimer’s disease continuum</span>. <em>Alzheimers. Dement.</em>, 19 (2), pp.658–670.
</div>
<div id="ref-Higgins2019-kn" class="csl-entry" role="listitem">
Higgins, J. P. T. (2019). <span class="nodecoration"><em>Cochrane handbook for systematic reviews of interventions</em></span>, Wiley cochrane series. 2nd ed. Higgins, J. and Thomas, J. (Eds). Hoboken, NJ : Wiley-Blackwell.
</div>
<div id="ref-Huang2023-vq" class="csl-entry" role="listitem">
Huang, L.-K., Kuan, Y.-C., Lin, H.-W. and Hu, C.-J. (2023). <span class="nodecoration">Clinical trials of new drugs for alzheimer disease: A 2020–2023 update</span>. <em>J. Biomed. Sci.</em>, 30 (1), p.83.
</div>
<div id="ref-Iaccarino2018-ta" class="csl-entry" role="listitem">
Iaccarino, L., Tammewar, G., Ayakta, N., Baker, S. L., Bejanin, A., Boxer, A. L., Gorno-Tempini, M. L., Janabi, M., Kramer, J. H., Lazaris, A., Lockhart, S. N., Miller, B. L., Miller, Z. A., O’Neil, J. P., Ossenkoppele, R., Rosen, H. J., Schonhaut, D. R., Jagust, W. J. and Rabinovici, G. D. (2018). <span class="nodecoration">Local and distant relationships between amyloid, tau and neurodegeneration in alzheimer’s disease</span>. <em>Neuroimage Clin</em>, 17, pp.452–464.
</div>
<div id="ref-Kovalchik2021-xq" class="csl-entry" role="listitem">
Kovalchik, S. (2021). <span class="nodecoration"><em><span>RISmed</span>: Download content from <span>NCBI</span> databases</em></span>.
</div>
<div id="ref-Martinelli2022-ic" class="csl-entry" role="listitem">
Martinelli, D. D. (2022). <span class="nodecoration">Evolution of alzheimer’s disease research from a health-tech perspective: Insights from text mining</span>. <em>International Journal of Information Management Data Insights</em>, 2 (2), p.100089.
</div>
<div id="ref-Nian2022-xw" class="csl-entry" role="listitem">
Nian, Y., Hu, X., Zhang, R., Feng, J., Du, J., Li, F., Bu, L., Zhang, Y., Chen, Y. and Tao, C. (2022). <span class="nodecoration">Mining on alzheimer’s diseases related knowledge graph to identity potential <span class="nocase">AD-related</span> semantic triples for drug repurposing</span>. <em>BMC Bioinformatics</em>, 23 (Suppl 6), p.407.
</div>
<div id="ref-Office_of_the_Commissioner2023-hu" class="csl-entry" role="listitem">
Office of the Commissioner. (2023). <span class="nodecoration"><em><span>FDA</span> grants accelerated approval for alzheimer’s disease treatment</em></span>. <a href="https://www.fda.gov/news-events/press-announcements/fda-grants-accelerated-approval-alzheimers-disease-treatment" class="uri">https://www.fda.gov/news-events/press-announcements/fda-grants-accelerated-approval-alzheimers-disease-treatment</a>; FDA.
</div>
<div id="ref-Silge2016-jf" class="csl-entry" role="listitem">
Silge, J. and Robinson, D. (2016). <span class="nodecoration"><em>Tidytext: Text mining and analysis using tidy data principles in <span>R</span></em></span>. 1.
</div>
<div id="ref-Villemagne2013-xk" class="csl-entry" role="listitem">
Villemagne, V. L., Burnham, S., Bourgeat, P., Brown, B., Ellis, K. A., Salvado, O., Szoeke, C., Macaulay, S. L., Martins, R., Maruff, P., Ames, D., Rowe, C. C., Masters, C. L. and Australian Imaging Biomarkers and Lifestyle (AIBL) Research Group. (2013). <span class="nodecoration">Amyloid <span class="math inline">\(\beta\)</span> deposition, neurodegeneration, and cognitive decline in sporadic alzheimer’s disease: A prospective cohort study</span>. <em>Lancet Neurol.</em>, 12 (4), pp.357–367.
</div>
<div id="ref-Wickham2019-rj" class="csl-entry" role="listitem">
Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L. D., François, R., Grolemund, G., Hayes, A., Henry, L., Hester, J., Kuhn, M., Pedersen, T. L., Miller, E., Bache, S. M., Müller, K., Ooms, J., Robinson, D., Seidel, D. P., Spinu, V., Takahashi, K., Vaughan, D., Wilke, C., Woo, K. and Yutani, H. (2019). <span class="nodecoration"><em>Welcome to the <span></span>tidyverse<span></span></em></span>. 4, p.1686.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>